src/components/Modals/DeployModal/DeployToWorld/DeployToWorld.tsx

```diff
@@ -0,0 +1,75 @@
+import { useCallback, useMemo, useState } from 'react'
+import { Button, SelectField } from 'decentraland-ui'
+import Modal from 'decentraland-dapps/dist/containers/Modal'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import Icon from 'components/Icon'
+import { Props } from './DeployToWorld.types'
+import styles from './DeployToWorld.module.css'
+
+const CLAIM_NAME_OPTION = 'claim_name_option'
+
+export default function DeployToWorld({ name, project, ensList, isLoading, onPublish, onClose, onBack }: Props) {
+  const [world, setWorld] = useState<string>()
+  const thumbnailUrl: string = project?.thumbnail || ''
+
+  const handlePublish = useCallback(() => {
+    if (world) {
+      onPublish(project?.id as string, world)
+    }
+  }, [onPublish, project, world])
+
+  const handleWorldSelected = useCallback((_, { value }) => {
+    if (value === CLAIM_NAME_OPTION) {
+      // TODO: Add claim name flow
+      return
+    }
+
+    setWorld(value)
+  }, [])
+
+  const worldOptions = useMemo(() => {
+    return [
+      ...ensList.map(ens => ({ text: ens.name, value: ens.name })),
```

Comment:
what do you think about using the `ens.subdomain` in the value?  the value will be `${ens.name}.dcl.eth`